pyproject.toml

```diff
@@ -48,3 +48,9 @@ module = [
     "requests_toolbelt"
 ]
 ignore_missing_imports = true
+
+[tool.yamlfix]
```

Comment:
this checks all .yamls in my .venv as well - which is undesirable

Same comment for ruff and mypy!

Reply:
not sure why, on my side it's doesn't include them... looks strange. however I can't find how to exclude folder in yamlfix it's only excludes specified files.